msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1524,6 +1528,131 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
 | **locale**| The locale of the message set by the client. |
 
+### Receive meeting participant events
+
+Your bot can receive real-time meeting events such as participant join and leave events through the meeting event subscription in Developer Portal. A bot receives the participant events only if subscribed to these events in Developer Portal.
+
+> [!NOTE]
+>
+> * Participant events are supported only for scheduled meetings.
+> * For a bot to receive participant events, ensure that you add the bot to the meeting before a participant joins or leaves the meeting.
+
+To subscribe to participant events, follow these steps:
+
+1. In [Developer Portal](https://dev.teams.microsoft.com/) open your bot app or import an existing app.
+1. In the **Meeting event subscriptions** section, select the events:
+    * Participant join
+    * Participant leave
+1. Select **Save**
+
+   :::image type="content" source="~/assets/images/apps-in-meetings/participant-events.png" alt-text="Screenshot shows how developer portal display for participant events.":::
+1. Ensure that the `OnlineMeetingParticipant.Read.Chat` RSC permission is configured in your app manifest.
+
+   If your app doesn't have the RSC permission, add it through the **Configure** > **Permissions** section of your app in Developer Portal. For more information, see [RSC permissions.](~/graph-api/rsc/resource-specific-consent.md)
+
+Following are the examples of the participant join and leave event payloads:
+
+# [Participant join event](#tab/participant-join-event) 
+
```

Comment:
I think we should add some info about what the code should look like, similar to start/end events.
This can come later if needed. 
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/95664393/5346b38b-9787-4a4d-a790-6507dbe5903f)

Reply:
Updated